content/events/2023-KCD-Munich

```diff
@@ -0,0 +1,13 @@
+title = "Kubernetes Community Day Munich"
+date = "2023-07-07T00:00:01Z"
+description = "Fermyon engineers Danielle Lancashire and Ryan Levick, along with Tom Gehrke from Liquid Reply, will be giving a \"Hands on with WebAssembly Microservices and Kubernetes\" tutorial at Kubernetes Community Day (KCD) Munich. Don't miss them on Monday July 17 @ 4 PM CEST as they help introduce attendees to WebAssembly."
```

Comment:
```suggestion
description = "Fermyon engineers Danielle Lancashire and Ryan Levick, along with Tom Gehrke from Liquid Reply, will be giving a \"Hands on with WebAssembly Microservices and Kubernetes\" tutorial at Kubernetes Community Day (KCD) Munich. Don't miss them on Monday July 17 @ 10:40 AM CEST as they help introduce attendees to WebAssembly."

As much as i wish it was 4pm, sadly it is not :(
```

Reply:
@endocrimes that last line isn't meant to be part of the suggestion patch right?!

Reply:
it was not 😂

Reply:
fixed!